bs-react-native-next/src/apis/Animated.md

```diff
@@ -244,10 +244,9 @@ external parallel:
 external stagger: (float, array(Animation.t)) => Animation.t = "";
 
 [@bs.module "react-native"] [@bs.scope "Animated"]
-external _loop: (Animation.t, {. "iterations": int}) => Animation.t = "loop";
-
-let loop = (~iterations=(-1), ~animation, ()) =>
-  _loop(animation, {"iterations": iterations});
+external loop:
+  (Animation.t, ~config: {. "iterations": int}=?, unit) => Animation.t =
```

Comment:
Thanks!

Two remarks:
1. Can we use `[@bs.obj]` for the config object (like for the other configs in this file and also following https://github.com/reasonml-community/bs-react-native/blob/master/bs-react-native-next/CONTRIBUTING.md#js-objects-passed-as-props)?

2. For easier usage, I think we should have `loop` with just the `Animation.t` parameter and `loopWithConfig` with the config parameter, too. Then the user does not need to remember the unit at the end.

Reply:
I am not sure about loop & loopWithConfig but I totally understand the reason.